### PR TITLE
Fixes #577: return apoc.nodes.count(['User'])

### LINKED ISSUE #577: return apoc.nodes.count(['User'])
Can we get a count of all nodes from the stats store? As we cannot currently do:
`RETURN count((:User)) AS uc, count((:User)-[:POLICY_HOLDER]->()) / count((:User)) as avg_policies`

### PR DESCRIPTION
**Pull request by vga91**
Fixes #577

### CODE REVIEW

File: docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.meta.nodes-lite.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+¦signature
+¦apoc.meta.nodes.count(nodes = [] :: LIST? OF STRING?, config = {} :: MAP?) :: (INTEGER?)
```
Comment (jexp) [inline docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.meta.nodes-lite.csv:2]: aren't these files generated and should not be checked in? everything in generated documentation.

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.meta.nodes.count.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+We can return all nodes with a label `MyCountLabel` or a label `ThirdLabel`
+
+[source,cypher]
+----
+RETURN apoc.meta.nodes.count(['MyCountLabel', 'ThirdLabel']) AS count;
+----
+
+.Results
+[opts="header"]
+|===
```
Comment (jexp) [inline docs/asciidoc/modules/ROOT/partials/usage/apoc.meta.nodes.count.adoc:19]: I don't think the implementation is what I expected it to be, I expected one result (row) per label

**Review by jexp — CHANGES_REQUESTED** [review]
I don't think the implementation is what I expected it to be, I expected one result (row) per label

**Comment by vga91** [comment]
> I don't think the implementation is what I expected it to be, I expected one result (row) per label

@jexp Sorry, I thought I answered 😅
I changed the implementation (1 row per label) and deleted some extra auto-generated files.

**Pull request merged**